iset.mm

```diff
@@ -65855,6 +65995,31 @@ the first case of his notation (simple exponentiation) and subscript it
   ctex $p |- ( A ~<_ _om -> A e. _V ) $=
     ( com cdom reldom brrelex1i ) ABCDE $.
 
+  ${
+    $d f A $.  $d f B $.  $d f F $.
+    $( The domain and range of a one-to-one, onto set function are
+       equinumerous.  This variation of ~ f1oeng does not require the Axiom of
+       Replacement nor the Axiom of Power Sets nor the Axiom of Union.
+       (Contributed by BTernaryTau, 7-Dec-2024.) $)
+    f1oen4g $p |- ( ( ( F e. V /\ A e. W /\ B e. X ) /\ F : A -1-1-onto-> B )
+        -> A ~~ B ) $=
+      ( vf wcel w3a wf1o wa cen wbr cv wex f1oeq1 spcegv imp 3ad2antl1 wb breng
+      3adant1 adantr mpbird ) CDHZAEHZBFHZIZABCJZKABLMZABGNZJZGOZUEUFUIUMUGUEUI
+      UMULUIGCDABUKCPQRSUHUJUMTZUIUFUGUNUEABGEFUAUBUCUD $.
+    $( $j usage 'f1oen4g' avoids 'ax-un'; $)
```

Comment:
'ax-rep' and 'ax-pow' shold be added (as mentioned in the comment).

Reply:
In set.mm, we have ``$( $j usage 'f1oen4g' avoids 'ax-pow' 'ax-rep' 'ax-un'; $)``.
And ~breng does not use ~ax-pow in set.mm.
So things are diffent in set.mm and iset.mm...

Reply:
Agreed, they are different. It would take someone who feels like investigating more to figure out where those differences are coming from. I will admit that my personal focus is mostly on having theorems with the same statements (including hypotheses and disjoint variables) in set.mm and iset.mm, because that is what lets us re-use proofs from one in the other.